crates/red_knot_python_semantic/src/types.rs

```diff
@@ -88,9 +88,64 @@ fn symbol_ty<'db>(db: &'db dyn Db, scope: ScopeId<'db>, name: &str) -> Type<'db>
         .unwrap_or(Type::Unbound)
 }
 
-/// Shorthand for `symbol_ty` that looks up a module-global symbol by name in a file.
+/// Return a list of the symbols that typeshed declares in the scope of `types.ModuleType`
+#[salsa::tracked(return_ref)]
+fn module_type_symbols<'db>(db: &'db dyn Db) -> Box<[Box<str>]> {
+    let Some(module_type) = KnownClass::ModuleType
+        .to_class(db)
+        .into_class_literal_type()
+    else {
+        return Box::default();
+    };
+
+    let module_type_scope = module_type.body_scope(db);
+    let module_type_symbol_table = symbol_table(db, module_type_scope);
```

Comment:
I don't understand enough about `ModuleType` to assess this and it might be a very dumb question so I'm just gonna raise it here and leave it up to you.

The symbol table only does lexicographical scoping, meaning it doesn't contain resolved entries from star imports. Is that a problem here or could it become a problem if typeshed adds such a star import?

Reply:
This query just returns a list of the symbols declared in the body of [this class in typeshed](https://github.com/python/typeshed/blob/c225ac758781c0599889b320c0a4f334366c5907/stdlib/types.pyi#L331); I don't think star imports are relevant here :-)